app/assets/javascripts/discourse/app/mixins/user-fields-validation.js

```diff
@@ -19,6 +19,7 @@ export default Mixin.create({
     let userFields = this.site.get("user_fields");
     if (userFields) {
       userFields = userFields
+        .filter((uf) => uf.requirement !== "for_existing_users")
```

Comment:
This user-fields-validation mixin is (IMO) very inappropriately coupled, but it is currently only used for sign-up validations, so we can add this condition here directly and have all sign-up paths work correctly without affecting anything else.